src/components/organisms/shared/OakPagination/OakPagination.test.tsx

```diff
@@ -71,14 +96,61 @@ describe("OakPagination Component", () => {
     expect(forwardsButton).toBeDisabled();
   });
 
+  it("handleChevronClick function", () => {
```

Comment:
it would be helpful to someone coming to this in the future for the spec to describe the desired behaviour and not the internal function. The test wouldn't make sense if someone changed the internal implementation.

Maybe:

```typescript
it('changes the page when the back and forward buttons are clicked')
```

Reply:
Changed ✅